### PR TITLE
Top graph in statistics

### PR DESCRIPTION
**Pull request by voidus**

### CODE REVIEW

**Comment by dsp** [comment]
The pull request looks good, nevertheless I think we need to make the line continuous, so we have to add vertical line segments that connect the current lines.

File: phprojekt/htdocs/phpr/StatisticsView.js [hunk 1]
```diff
@@ -105,16 +107,47 @@ define([
...
+                if (locale.isWeekend(date)) {
+                    return heightForTimebars;
+                }
+                return heightForMinutesToWork;
+            };
+
+            // horizontal lines
+            svgData.enter()
+                .append("svg:line")
+                    .attr("x1", function(d, i) {
```
Comment (exi) [inline phprojekt/htdocs/phpr/StatisticsView.js:123]: wrong quotations marks from here on down, use ' and not "

Comment (voidus) [inline phprojekt/htdocs/phpr/StatisticsView.js:123]: Has been moved in the refactoring branch, fixed there.

Comment (exi) [inline phprojekt/htdocs/phpr/StatisticsView.js:123]: does that mean that this pull request can be closed?

Comment (voidus) [inline phprojekt/htdocs/phpr/StatisticsView.js:123]: Yes, it can'

edit: No, it can't'

**Pull request merged**